content/docs/intro/vs/cloud_templates.md

```diff
@@ -11,18 +10,26 @@ menu:
 aliases: ["/docs/reference/vs/cloud_templates/"]
 ---
 
+What is Pulumi?
```

Comment:
Leading off with what sounds like an ad for Pulumi doesn't seem like the right thing, here, since these pages are aimed at delivering comparative info, and they'd seem to be working relatively well in that regard. I'd remove these headings and this introductory section and leave these pages structurally as they are today, unless breaking them up is solving a problem that can't be solved otherwise.

Reply:
It was mostly to get the "What is CloudFormation?" part in there, as that is the first snippet question on the SERP, and CloudFormation is the primary KW. The "What is Pulumi?" part was more to balance it out and provide a clearer structure, since this is a comparison page. Would it be ok to switch the order and lead with CloudFormation?